apps/docs/docs/reference/ui/molecules/ProductCard.mdx

```diff
@@ -0,0 +1,181 @@
+---
+description: Card with summarized information about a product.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+---
+
+import PropsSection from "@site/src/components/PropsSection/PropsSection";
+
+ProductCard displays summarized information about a product.
+
+The `ProductCard` is a compound of the following:
+- `ProductCardImage`: wraps the product's image.
+- `ProductCardContent`: wraps the content's details.
+- `ProductCardActions`: wraps the call-to-action button of the product card.
+
+## Import
+
+```tsx
+import {
+  ProductCard,
+  ProductCardActions,
+  ProductCardContent,
+  ProductCardImage,
+} from '@faststore/ui'
+```
+
+## Usage
+
+### Default
+
+```tsx live
+ <LocationProvider>
```

Comment:
Mari, not sure about the reason, but can you check this `Provider` usage.

![image](https://user-images.githubusercontent.com/3356699/171950642-8ceb5c13-3703-40ee-878f-2d3cb410c035.png)

Reply:
Thanks for pointing that out, Fanny! I'm going to check it here.